pkg/cli/cmd/run/run.go

```diff
@@ -159,28 +159,56 @@ func (r *Runner) Run(ctx context.Context) error {
 		return clierrors.Message("Only kubernetes runtimes are supported.")
 	}
 
-	// We start three background jobs and wait for them to complete.
+	applicationSelector, err := portforward.CreateLabelSelectorForApplication(r.ApplicationName)
+	if err != nil {
+		return err
+	}
+
+	dashboardSelector, err := portforward.CreateLabelSelectorForDashboard()
+	if err != nil {
+		return err
+	}
+
+	// We start 5 background jobs and wait for them to complete.
 	group, ctx := errgroup.WithContext(ctx)
 
-	// 1. Display port-forward messages
-	status := make(chan portforward.StatusMessage)
+	// 1. Display port-forward messages for application
+	applicationStatusChan := make(chan portforward.StatusMessage)
+	group.Go(func() error {
+		r.displayPortforwardMessages(applicationStatusChan)
+		return nil
+	})
+
+	// 2. Display port-forward messages for dashboard
+	dashboardStatusChan := make(chan portforward.StatusMessage)
 	group.Go(func() error {
-		r.displayPortforwardMessages(status)
+		r.displayPortforwardMessages(dashboardStatusChan)
 		return nil
 	})
 
-	// 2. Port-forward
+	// 3. Port-forward application
 	group.Go(func() error {
 		return r.Portforward.Run(ctx, portforward.Options{
-			ApplicationName: r.ApplicationName,
-			Namespace:       namespace,
-			KubeContext:     kubeContext,
-			StatusChan:      status,
-			Out:             os.Stdout,
+			LabelSelector: applicationSelector,
+			Namespace:     namespace,
+			KubeContext:   kubeContext,
+			StatusChan:    applicationStatusChan,
+			Out:           os.Stdout,
+		})
+	})
+
+	// 4. Port-forward dashboard
+	group.Go(func() error {
+		return r.Portforward.Run(ctx, portforward.Options{
+			LabelSelector: dashboardSelector,
+			Namespace:     "radius-system",
```

Comment:
does it make sense to hardcode this namespace?

Reply:
not really, fixed